packages/replayio/src/utils/protocol/api/addOriginalSource.ts

```diff
@@ -0,0 +1,14 @@
+import { addOriginalSourceParameters, addOriginalSourceResult } from "@replayio/protocol";
+import ProtocolClient from "../ProtocolClient";
+
+export async function addOriginalSource(
+  client: ProtocolClient,
+  params: addOriginalSourceParameters
+) {
+  await client.waitUntilAuthenticated();
+
+  return await client.sendCommand<addOriginalSourceParameters, addOriginalSourceResult>({
+    method: "Record.addOriginalSource",
```

Comment:
https://static.replay.io/protocol/tot/Recording/#method-addOriginalSource
```suggestion
    method: "Recording.addOriginalSource",
```